Integrations/python/deephaven/KafkaTools/__init__.py

```diff
@@ -198,6 +314,31 @@ def getAvroSchema(schemaServerUrl, resourceName, version):
     return _java_type_.getAvroSchema(schemaServerUrl, resourceName, version)
 
 
+@_passThrough
+def ignoreSpec():
+    """
+    :return: io.deephaven.kafka.KafkaTools.KeyOrValueSpec
+    """
+    
+    return _java_type_.ignoreSpec()
+
+
+@_passThrough
+def jsonSpec(*args):
```

Comment:
When do we expect these to be used instead of the friendly ones above?

Reply:
They shouldn't, I am missing listing them for ignoring on PythonStaticGenerator, thanks.